cmd/main.go

```diff
@@ -111,7 +113,7 @@ func handle(logger *zap.Logger) {
 
 	statUtil := &(driver.VolumeStatUtils{})
 
-	err = ibmCSIDriver.SetupIBMCSIDriver(ibmcloudProvider, mounter, statUtil, nil, &nodeInfo, logger, csiConfig.CSIDriverName, vendorVersion)
```

Comment:
I am assuming, it would be better to take this as an env variable rather than a flag.

Reply:
Instead of passing extra parmeters here, it would be good to check and initialise directly in the SetupIBMCSIDriver, or NewNodeServer function itself.